### ISSUE TITLE
model name '../data/bert_models/chinese_finetuned_lm/pytorch_model.bin' was not found in model name list . Creating an empty model card.

### ISSUE THREAD

**Issue by zbbwss** [huggingface/transformers#3549]
nlp = pipeline('fill-mask',
                   # model=args.bert_model_path,
                   # config=args.bert_config_path,
                   # tokenizer=args.bert_model_dir
                    model = '../data/bert_models/chinese_finetuned_lm/pytorch_model.bin',
                    config = '../data/bert_models/chinese_finetuned_lm/config.json',
                    tokenizer = "../data/bert_models/chinese_finetuned_lm/"
                   )
i use fine-tune model (chinese bert) ,when i end fine-tune  i can not load model  !
